packages/framework/src/Controller/Admin/SalesRepresentativeController.php

```diff
@@ -64,7 +66,7 @@ public function listAction(): Response
     #[Route(path: '/sales-representative/new/')]
     public function newAction(Request $request): Response
     {
-        $salesRepresentativeData = new SalesRepresentativeData();
+        $salesRepresentativeData = $this->salesRepresentativeDataFactory->create();
```

Comment:
ah, here is the factory, awesome :ok_hand:

Reply:
indeed indeed